main/sdr.go

```diff
@@ -62,7 +62,7 @@ type Dump1090TermMessage struct {
 func (e *ES) read() {
 	defer e.wg.Done()
 	log.Println("Entered ES read() ...")
-	cmd := exec.Command(STRATUX_HOME + "/bin/dump1090", "--oversample", "--net-stratux-port", "30006",  "--net", "--device-index", strconv.Itoa(e.indexID), "--ppm", strconv.Itoa(e.ppm))
+	cmd := exec.Command(STRATUX_HOME + "/bin/dump1090", "--gain", "37.2", "--net-stratux-port", "30006",  "--net", "--device-index", strconv.Itoa(e.indexID), "--ppm", strconv.Itoa(e.ppm))
```

Comment:
Have you tested this? I did a while ago with different antenna configurations and found changing the gain levels to be ineffective, the frontend on the SDR is overloaded before the gain levels make a difference. Beyond 100 ft with a high gain antenna and the highest gain level didn't cause reception issues.

Reply:
We had reports of 1090ES planes getting lost when coming too close and @VirusPilot did some tests today with his 250W transponder and yes, it did indeed make quite a difference for receiving close signals.
With dump1090's dev branch (which now has optional adaptive gain), this can be tested quite nicely, as it will tune down gain to receive close aircraft - and there is indeed a strong correlation between distance to the transponder and the gain level that dump1090 determines to be appropriate to properly decode it.

Considering the variance in SDR quality/sensitivity and antennas, maybe the SDR you tested with was just one of the "more deaf" ones?

Reply:
"Deaf" wouldn't cause that behavior, I'm more referencing gain compression.

https://github.com/cyoung/stratux/issues/203#issuecomment-214565215

I can't get my own transponder signal in my aircraft using any R820T(2) receivers (I've tested them all), so I'd guess the new adaptive gain on dump1090 ends up pulling the gain back pretty far permanently when used inside a small aircraft with a transponder installed.

Reply:
Yes, own transponder is certainly another level, VirusPilot tried on the ground, testing how far he needs to walk away to receive it. And yes, the adaptive gain is not usuable if you carry a transponder very close - but it is usuable for testing if reducing gain helps at all (and what gain works at what distance). @VirusPilot measured something like this:
- With 50db gain reception fails below 250m
- With 37.5db gain below 150m
- With 30db gain below 100m

Since 100m is certainly too low to react anyway, and we didn't want to break anything for people with minimalistic internal antennas, we kinda settled with "something around 35" by pure guessing. Not too scientific admittedly..